aspnetcore/blazor/debug.md

```diff
@@ -210,6 +210,20 @@ For guidance on configuring VS Code assets in the `.vscode` folder and where to
 >
 > You can't automatically rebuild the backend **`Server`** app of a hosted Blazor WebAssembly solution during debugging, for example by running the app with [`dotnet watch run`](xref:tutorials/dotnet-watch).
 
+To debug a **published**, hosted Blazor WebAssembly app, configure debugger support (`DebuggerSupport`) and copy output symbols to the `publish` directory (`CopyOutputSymbolsToPublishDirectory`) in the **`Client`** app's project file:
+
+```xml
+<DebuggerSupport>true</DebuggerSupport>
+<CopyOutputSymbolsToPublishDirectory>true</CopyOutputSymbolsToPublishDirectory>
+```
+
+By default, publishing an app disables the preceding properties by setting them to `false`.
+
+> [!WARNING]
+> Published, hosted Blazor WebAssembly apps should only enable debugging when deploying published assets ***locally***. Do **not*** deploy a published app into production with the `DebuggerSupport` property set to `true`.
+>
+> If `CopyOutputSymbolsToPublishDirectory` is set to `true` in a published, production app, there are no security risks, but app size and deployment speed are adversely impacted for no benefit. Therefore, apps deployed into production should ***not** enable copying symbols to the `publish` directory.
```

Comment:
> If `CopyOutputSymbolsToPublishDirectory` is set to `true` in a published, production app, there are no security risks

I dunno if I feel confident making this statement. It feels like we should generally not recommend having symbols on servers for apps, even if they are not served by the Blazor front-end.

Reply:
It's worded to not suggest it, but we've been ultra conservative before in similar scenarios, so I'll update it.